themes/bootstrap3/js/doi.js

```diff
@@ -49,12 +49,13 @@ VuFind.register('doi', function Doi() {
   function init(_container) {
     var container = _container || $('body');
```

Comment:
Here is also a reference to jquery object.

Reply:
jQuery removed here.